src/proofs/proof.v3.controller.ts

```diff
@@ -23,12 +22,4 @@ export class ProofsV3Controller {
 
     return this.proofsService.getBouncerProof(Chain.getChain(chain), address);
   }
-
-  @Get('/citadel')
```

Comment:
these removals on v2 and v3 are ok as they were never productionized and therefore do not constitute any form of removal or service or removal of backward compatibilities